main/modified.tmLanguage.json

```diff
@@ -3236,7 +3263,7 @@
             "name": "meta.function.objc",
             "patterns": [
                 {
-                    "begin": "(\\()",
+                    "begin": "(?<=(?:-|\\+)\\s*)(\\()",
```

Comment:
Note: this breaks the parser (groups are not allowed inside lookbehinds) so I had to remove this when publishing the `0.2.0` version

Reply:
Does

"(?<=-|\\+\\s*)(\\()" work instead?

Reply:
I don't think it allows quantified values either, but I'd have to check to be sure